src/protocols/ShortcutsInhibit.cpp

```diff
@@ -73,6 +73,9 @@ bool CKeyboardShortcutsInhibitProtocol::isInhibited() {
     if (!g_pCompositor->m_pLastFocus)
         return false;
 
+    if (g_pCompositor->getWindowFromSurface(g_pCompositor->m_pLastFocus.lock())->m_sWindowData.noShortcutsInhibit.valueOrDefault())
```

Comment:
ehhhh is this a good idea?

Reply:
yeah you're probably right
what's the solution? storing a `PHLWINDOWREF` on each `CWLSurface` when created or will that cause issues?

Reply:
not my point, the question is should the state of isInhibited() be altered by the focus. IMO no

Reply:
it already is doe
the next bit of the function is
```cpp
for (auto& in : m_vInhibitors) {
     if (in->surface() != g_pCompositor->m_pLastFocus)
         continue;
```
which already depends on focus

Reply:
ah, right. It's fine then